tox.ini

```diff
@@ -0,0 +1,40 @@
+[tox]
+envlist = py39,py310
+
+[testenv]
+deps=-rtest-requirements.txt
+commands=pytest -v {posargs}
+whitelist_externals=sh
+
+[testenv:static]
+deps=
+	-rtest-requirements.txt
+	black==21.12b0
+	pylint==2.12.2
+commands=
+	black --check .
+	sh -c 'pylint ubi_manifest; test $(( $? & (1|2|4|32) )) = 0'
+
+[testenv:cov]
+deps=
+	-rtest-requirements.txt
+	pytest-cov
+usedevelop=true
+commands=
+	pytest --cov-report=html --cov-report=xml --cov=ubi_manifest --cov-fail-under=100 {posargs}
+
+[testenv:docs]
+deps=
+	sphinx
+	alabaster
+use_develop=true
+commands=
+	sphinx-build -M html docs docs/_build -W
+
+[testenv:pidiff]
```

Comment:
As ubi-manifest is a web service and not a library (right?), this tool probably doesn't make sense.  It's meant for checking the API of a library.

Reply:
yes, it's a service, I guess this was just copy-paste issue.